Fix build failure:
  CXXLD    fb-gnash
/usr/bin/ld: fb_gnash-gnash.o: undefined reference to symbol '_ZN5boost6system15system_categoryEv'
/usr/bin/ld: note: '_ZN5boost6system15system_categoryEv' is defined in DSO /lib64/libboost_system-mt.so.1.50.0 so try adding it to the linker command line
/lib64/libboost_system-mt.so.1.50.0: could not read symbols: Invalid operation
collect2: error: ld returned 1 exit status

Index: gnash-0.8.10/macros/boost.m4
===================================================================
--- gnash-0.8.10.orig/macros/boost.m4
+++ gnash-0.8.10/macros/boost.m4
@@ -37,7 +37,7 @@ AC_DEFUN([GNASH_PATH_BOOST],
   boost_headers="detail/lightweight_mutex.hpp thread/thread.hpp multi_index_container.hpp multi_index/key_extractors.hpp thread/mutex.hpp program_options/options_description.hpp iostreams/stream.hpp"
   dnl this is a list of *required* libraries. If any of these are missing, this
   dnl test will return a failure, and Gnash won't build.
-  boost_libs="thread program_options iostreams"
+  boost_libs="thread program_options iostreams system"
 
   dnl this is a list of *recommended* libraries. If any of these are missing, this
   dnl test will return a warning, and Gnash will build, but testing won't work.
